src/app/server/CommissioningWindowManager.cpp

```diff
@@ -354,7 +354,14 @@ CHIP_ERROR CommissioningWindowManager::StartAdvertisement()
 #if CONFIG_NETWORK_LAYER_BLE
     if (mIsBLE)
     {
-        ReturnErrorOnFailure(chip::DeviceLayer::ConnectivityMgr().SetBLEAdvertisingEnabled(true));
+        CHIP_ERROR err = chip::DeviceLayer::ConnectivityMgr().SetBLEAdvertisingEnabled(true);
+        // BLE advertising may just not be supported.  That should not prevent
+        // us from opening a commissioning window and advertising over IP.
+        if (err == CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE)
+        {
+            err = CHIP_NO_ERROR;
```

Comment:
Should we warn here? Or is just switch to ok fine since UNSUPPORTED_FEATURE is very specific?

I am surprised we would have `CONFIG_NETWORK_LAYER_BLE` yet advertisement not supported.

Reply:
As I described in the PR, this is a case of the library being built with BLE for use as a commissioner (so it needs BLE discovery), but without advertisement support because commissioners never need to advertise over BLE, and the same library being used for all example apps.

We could add a log, though, to help debuggability.